homeassistant/components/zwave/const.py

```diff
@@ -324,14 +324,30 @@
 DISC_GENRE = "genre"
 DISC_INDEX = "index"
 DISC_INSTANCE = "instance"
-DISC_LABEL = "label"
 DISC_NODE_ID = "node_id"
 DISC_OPTIONAL = "optional"
 DISC_PRIMARY = "primary"
-DISC_READONLY = "readonly"
+DISC_SCHEMAS = "schemas"
 DISC_SPECIFIC_DEVICE_CLASS = "specific_device_class"
 DISC_TYPE = "type"
 DISC_VALUES = "values"
-DISC_WRITEONLY = "writeonly"
 
-METER_RESET_INDEX = 33
+INDEX_ALARM_TYPE = 0
```

Comment:
I'll add links in a bit. I debated myself a bit on whether we should add links, since the links could become broken in the future, but I guess it's better than nothing at all.

As for the indexes, they are actually explicitly defined for this purpose by open-zwave. I couldn't find an answer in the documentation, but there is an answer in the thread linked below, and they are defined explicitly as enums in the source:

https://groups.google.com/forum/#!topic/openzwave/tqxQr5XbG2Q
https://github.com/OpenZWave/open-zwave/blob/master/cpp/src/command_classes/SensorMultilevel.cpp#L50